lib/kwtool_web/controllers/api/v1/session_controller.ex

```diff
@@ -0,0 +1,21 @@
+defmodule KwtoolWeb.Api.V1.SessionController do
+  use KwtoolWeb, :controller
+
+  alias Kwtool.Account.{Guardian, Users}
+  alias KwtoolWeb.Api.V1.ErrorView
+
+  def create(conn, %{"email" => email, "password" => password}) do
+    case Users.authenticate_user(email, password) do
+      {:ok, user} ->
+        {:ok, jwt, _full_claims} = Guardian.encode_and_sign(user, %{})
+
+        render(conn, "show.json", %{data: %{id: user.id, email: user.email, jwt: jwt}})
```

Comment:
For the ID of login resource, maybe we can generate some value e.g. timestamps.

Reply:
Updated in https://github.com/byhbt/kwtool/pull/45/commits/d4700d1b319c67aa78cd2b7a4e64e1e10af3378a